From 408c3757279c999a32e454be9de1216ac4969849 Mon Sep 17 00:00:00 2001
From: Stefan Haller <stefan@haller-berlin.de>
Date: Tue, 2 Sep 2025 13:25:36 +0200
Subject: [PATCH 17/18] gitk: Avoid horizontal scrolling of the commit list and
 diff

With two-finger scrolling on a track pad, horizontal scrolling of the commit
list is usually unintentional, so avoid it completely. It should still be
possible to scroll horizontally using shift-wheel with an actual mouse, but I
couldn't test this because I don't have one.
---
 gitk-git/gitk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/gitk-git/gitk b/gitk-git/gitk
index 27ee5e8c54..92f3932d9d 100755
--- a/gitk-git/gitk
+++ b/gitk-git/gitk
@@ -2354,7 +2354,6 @@ proc bind_mousewheel {} {
         bind $ctext <TouchpadScroll> {
             lassign [tk::PreciseScrollDeltas %D] deltaX deltaY
             $ctext yview scroll [precisescrollval $deltaY 2] units
-            $ctext xview scroll [precisescrollval $deltaX 2] units
         }
         bind $cflist <TouchpadScroll> {
             lassign [tk::PreciseScrollDeltas %D] deltaX deltaY
@@ -2362,7 +2361,6 @@ proc bind_mousewheel {} {
         }
         bind $canv <TouchpadScroll> {
             lassign [tk::PreciseScrollDeltas %D] deltaX deltaY
-            $canv xview scroll [precisescrollval $deltaX] units
             allcanvs yview scroll [precisescrollval $deltaY] units
         }
     }
-- 
2.52.0

